Sources/UITouch.swift

```diff
@@ -53,6 +53,8 @@ public class UITouch {
             action(recognizer)
         }
     }
+
+    var hasBeenCancelledByAGestureRecognizer = false
```

Comment:
please add an explicit `internal` here, as in `internal var` - we are always (hopefully) explicit about this in the UIKit project to avoid confusion and make our intent clear

Reply:
done!